Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,413 @@
+    , sosURLPathName = arg // URL. Represents the URL path name.
+    , sosFileIdentifier = fileid // File hash or URL
+    , sosDPIInspectedFlow = dpi // Indicates a flow was inspected by DPI. Applies only to Connection Closed messages.
+    , DstNatPortNumber = dnpt
+    , SrcNatPortNumber = snpt
+    , sosBladeID = bid // Blade ID
+    , sosUUID = uuid
+    , sosHttpRequestMethod = HttpRequestMethod
+    , sosFileName = FileName
+    , DvcOriginalAction = fw_action
+  | extend
+    ThreatName = coalesce(sosASWSignatureName, sosGAVSignatureName, sosIPSSignatureName, "")
+    , ThreatId = coalesce(sosAppSignatureID, "")
+    , ThreatCategory = coalesce(sosIPSCategoryName, sosAntiSpywareCategory, "")
+    , DstNatPortNumber = toint(DstNatPortNumber)
+    , SrcNatPortNumber = toint(SrcNatPortNumber)
+  | extend AdditionalFields = bag_pack(
+    "ReceivedPackets", sosReceivedPackets
+    , "SentPackets", sosSentPackets
+    , "ConnectionDuration", sosConnectionDuration
+    , "AppRulePolicyId", sosAppRulePolicyId
+    , "AppRulePolicyName", sosAppRulePolicyName
+    , "AppRuleService", sosAppRuleService
+    , "AppRuleType", sosAppRuleType
+    , "AppRuleObject", sosAppRuleObject
+    , "AppRuleObjectContent", sosAppRuleObjectContent
+    , "AppRuleAction", sosAppRuleAction
+    , "AppIDNumber", sosAppIDNumber
+    , "AppID", sosAppID
+    , "AppCategoryID", sosAppCategoryID
+    , "AppSignatureID", sosAppSignatureID
+    , "IPSCategoryName", sosIPSCategoryName
+    , "AntiSpywareCategory", sosAntiSpywareCategory
+    , "URLPathName", sosURLPathName
+    , "FileIdentifier", sosFileIdentifier
+    , "DPIInspectedFlow", sosDPIInspectedFlow
+    , "BladeID", sosBladeID
+    , "UUID", sosUUID
+    , "HttpRequestMethod", sosHttpRequestMethod
+    , "FileName", sosFileName
+    , "FileSize", FileSize
+    , "HttpReferrer", sosHttpReferrer
+    , "HttpUserAgent", HttpUserAgent
+    , "CaptureATPVerdict", sosCaptureATPVerdict
+    , "ApplicationID", sosApplicationID
+    , "CFSCategoryID", sosCFSCategoryID
+    , "CFSCategoryName", sosCFSCategoryName
+    , "CFSPolicyName", sosCFSPolicyName
+    , "AppControlFileName", sosAppControlFileName
+    , "AppCategoryName", sosAppCategoryName
+    , "AppIDName", sosAppIDName
+    , "AppSignatureName", sosAppSignatureName
+    , "IPSFullString", sosIPSFullString
+    , "IPSSignatureName", sosIPSSignatureName
+    , "LegacyMessageCategory", sosLegacyMessageCategory
+    , "LogMsgCategory", sosLogMsgCategory
+    , "LogMsgNote", sosLogMsgNote
+    , "LogMsgSeverity", sosLogMsgSeverity
+    , "SourceVPNPolicyName", sosSourceVPNPolicyName
+    , "DestinationVPNPolicyName", sosDestinationVPNPolicyName
+    , "SerialNumber", sosSerialNumber
+    , "DeviceModel", sosDeviceModel
+    , "LogMsgID", sosLogMsgID
+    , "RequestURL", RequestURL
+    , "Activity", EventMessage
+    , "EventMessageDetail", sosEventMessageDetail
+    , "UserSessionType", sosUserSessionType
+    , "UserSessionDuration", sosUserSessionDuration

```

Comment:
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/f2b4d7ee-7d5c-4589-817e-41b1da8844da)

Reply:
I've found that 'DeviceCustomString6Label' (the User Session duration) probably won't be populated in any messages returned by the Network Session or Web Session parsers. Based on some queries on CommonSecurityLog, I've only seen this field populated on messages in categories that are not included in either Network Session or Web Session parsers. I will remove it.